### PR TITLE
feat(client): upgrade shell scripts and replace docker image links

### PR DESCRIPTION
**Pull request by mask-pp**

### CODE REVIEW

**Comment by davidtaikocha** [comment]
Looks like the PR title lint is failed, right now we follow: https://www.conventionalcommits.org/en/v1.0.0/ for our PR title :)

File: integration_test/nodes/init.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ DIR=$(
 )
 
 # Download solc for PlonkVerifier
-$TAIKO_MONO_DIR/packages/protocol/script/download_solc.sh
+#$TAIKO_MONO_DIR/packages/protocol/script/download_solc.sh
```
Comment (davidtaikocha) [inline integration_test/nodes/init.sh:11]: Let's just delete it?

File: integration_test/nodes/docker-compose.yml [hunk 2]
```diff
@@ -2,27 +2,27 @@ version: "3.9"
...
+      - "18545:8545"
+      - "18546:8545"
     entrypoint:
       - anvil
       - --host
       - "0.0.0.0"
 
   l2_execution_engine:
-    image: gcr.io/evmchain/taiko-geth:taiko
+    image: gcr.dockerproxy.com/evmchain/taiko-geth:taiko
```
Comment (davidtaikocha) [inline integration_test/nodes/docker-compose.yml:17]: Not sure if this will slow down those who are not in China?

File: scripts/common.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  fi
+}
+
+check_command() {
+  local command="$1"
+
+  if ! command -v "$command" &> /dev/null; then
+  print_error "$command could not be found"
+  exit
+fi
```
Comment (davidtaikocha) [inline scripts/common.sh:27]: looks like we have some indentation issues here?

File: integration_test/nodes/init.sh (hunk 1 above)
Comment (mask-pp) [reply integration_test/nodes/init.sh:11]: Has given a check in entrypoint.sh about solc tool.

File: integration_test/nodes/docker-compose.yml (hunk 2 above)
Comment (mask-pp) [reply integration_test/nodes/docker-compose.yml:17]: It's very fast not in China.

File: scripts/common.sh (hunk 3 above)
Comment (mask-pp) [reply scripts/common.sh]: fixed.

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**